lib/charms/mysql/v0/backups.py

```diff
@@ -355,6 +364,25 @@ def _on_create_backup(self, event: ActionEvent) -> None:
         })
         self.charm._on_update_status(None)
 
+    def _can_cluster_perform_backup(self, force: bool) -> tuple[bool, str | None]:
+        """Validates whether this cluster can perform a backup.
+
+        Returns: tuple of (success, error_message)
+        """
+        cluster_status = self.charm._mysql.get_cluster_status()
```

Comment:
Force should take precedence (or be assessed out of the function).
The point is, if we force we bypass all health checking.

And maybe we should rename the parameter to something like `skip_health_check` for clear intention?